gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_8_0/01_add_portal_notifications_groups_table.yml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.8.0_01_add_portal_notifications_groups_table_1
+      author: GraviteeSource Team
+      changes:
+        - createTable:
+            tableName: ${gravitee_prefix}portal_notification_config_groups
+            columns:
+              - column: {name: user, type: nvarchar(64), constraints: { nullable: false } }
+              - column: {name: reference_id, type: nvarchar(64), constraints: { nullable: false } }
+              - column: {name: reference_type, type: nvarchar(64), constraints: { nullable: false } }
+              - column: {name: group, type: nvarchar(64), constraints: { nullable: false } }
+        - addPrimaryKey:
```

Comment:
This line should trigger our [dangerjs script](https://github.com/gravitee-io/gravitee-api-management/blob/master/.circleci/danger/dangerfile.js#L21) I think. 🤔 

In the past I remember that we had an issue with a customer who shared with us that it was not possible for him to start the project because it can be forbidden to create a table without primary key. So this dangerjs script has been added to ensure that we create the table with the PK directly. do you think it could be possible here ? 

NB: Here is the old jira ticket https://gravitee.atlassian.net/browse/APIM-1978